project4/project/ebs/views.py

```diff
@@ -0,0 +1,263 @@
+                return HttpResponse(json.dumps(response), 
+                                    content_type='application/json')
+
+            elif password!=password1:
+                response = {'status': 'Error',
+                            'message': "please fill the details"}
+                return HttpResponse(json.dumps(response), 
+                                    content_type='application/json')
+            obj = forgotpassword.objects.get(activation_key=hash1)
+            user = User.objects.get(username=obj.username)
+            user.set_password(password)
+            user.save()
+            forgotpassword.objects.get(id=obj.id).delete()
+            response = {'status': 'success',
+                        'message': "password updated successfully"}
+            return HttpResponse(json.dumps(response), 
+                                content_type='application/json')
+
+    except Exception as e:
+        response = {'status': 'Error', 'message': 'invalid link or token has been expired.'}
+        return HttpResponse(json.dumps(response), 
+                            content_type='application/json')
+
+def create_blog(request):
+
+        if request.method=='POST':
+            import pdb
+            pdb.set_trace()
+            blogform=BlogForm(request.POST or None,request.FILES or None)
+            blogfileform=BlogFileForm(request.POST or None,request.FILES or None)
+            if blogform.is_valid() and  blogfileform.is_valid():
+                files = request.FILES.values()
+                blog=blogform.save(commit=False)
+                orgobj=Organisation.objects.get(orgname='Oyo')
+                blog.organisation_id=orgobj.id
+                if 'button1' in request.POST:
+                    blog.draft=True
+                blog.save()
+                for a_file in files:
+                    instance=BlogFile()
+                    instance.blog_id=blog.id
+                    instance.attachments=a_file
+                    instance.save()
+                blog.save()
+
+                """instance=Blog()
+                instance.title=blogform.cleaned_data['title']
+                instance.description=blogform.cleaned_data['description']
+                instance.attachments=a_file
+                instance.tags=blogform.cleaned_data['tags']
+                instance.catagories=blogform.cleaned_data['catagories']
+                instance.commentstate=blogform.cleaned_data['commentstate']
+                instance.published=blogform.cleaned_data['published']
+                orgobj=Organisation.objects.get(orgname='Oyo')
+                instance.organisation_id=orgobj.id
+                instance.save()"""
+                return HttpResponse("success")
+        else:
+            blogfileform=BlogFileForm()
+            blogform=BlogForm()
+            return render(request,'ebs/create_blog.html',{'blogform': blogform,'blogfileform':blogfileform})
```

Comment:
Should create separate file for each module/feature, all should not be included in view.py file. Need to create new task in next sprint.